tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -380,6 +380,28 @@ tensorflow::Status CreateBroadcastableScalarConstant(
   return Status::OK();
 }
 
+// Convert an axis from TF format to TRT format while validating.
```

Comment:
Yes! Should I update the other converters in this PR, or later once this is merged?

For ExpandDims the bounds are slightly different: [0, size] instead of [0, size). Not sure how to handle that slight difference cleanly. But I think the rest of the ops would use the same bounds checking. Maybe another arg `check_bounds` to enable bounds checking, and ExpandDims could handle the special bounds itself?

Reply:
I am fine with adding a TODO for now. Did not notice that bounds for ExpandDims is different. your proposal sounds good to me.